packages/vanilla-components/src/components/referral-list/referral-list.tsx

```diff
@@ -105,7 +105,23 @@ export class ReferralList {
     if (this.referrals) {
       referralsRow = (
         this.referrals.map((ref) => {
-          const referraltype = ref.rewards.length > 0 ? 'converted' : 'pending'
+          const referraltype = ref.rewards.length > 0 ? 'converted' : 'pending';         
+          const isCancelled = ref.rewards.length > 0 && ref.rewards[0].statuses && ref.rewards[0].statuses.indexOf("CANCELLED") > -1;
+
+          if (ref.rewards.length === 1 && isCancelled) return;
+
+          // Removes cancelled rewards from rewards array so that multiple rewards don't show cancelled amoungst them
+          const moreThanOneReward = ref.rewards.length > 1;
```

Comment:
you could remove this variable, and use `ref.rewards.length > 1` in the if statement.